features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2344,4 +2344,9 @@
         {% endfor %}
     </ReverseProxyConfig>
     {% endif %}
+
+    <!-- Configuration for preserving the current session and token when updating the password. -->
+    <PasswordUpdate>
+        <PreserveLoggedInSession>{{password_update.preserve_logged_in_session}}</PreserveLoggedInSession>
```

Comment:
[password_update] should not be a top level configuration. Can you check the existing configurations and come up with a proper existing top level config

Reply:
Re-defined under [identity_mgt] configurations